src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/jpa/entity/PeriodLengthEntity.kt

```diff
@@ -95,9 +94,6 @@ class PeriodLengthEntity(
     weeks = periodLength.periodWeeks
     days = periodLength.periodDays
     periodLengthType = type
-    // TODO not sure if the status can change via the update-period-length legacy route - to check with syscon
```

Comment:
Update of a period length doesnt change the status so this has been removed - it can be changed by the updateSentence route thougg